data/json/items/ammo/303.json

```diff
@@ -0,0 +1,110 @@
+[
+  {
+    "id": "303_bthp",
+    "type": "AMMO",
+    "name": { "str_sp": ".303 British BTHP" },
+    "description": "Modern manufactured .303 caliber cartridges, loaded with 174gr boat-tailed hollow-point bullets.  Starting life as a black powder cartridge in service with British Long Lee rifles in the early nineteenth century, the cartridge evolved into a center-fired smokeless-powder design and became the standard rifle and machinegun cartridge of the British commonwealth.  The round’s boat tail design gives it greater stability with lese aerial drag over long ranges, making it favorable for competition shooting, whilst the bullet’s hollow-point construction leaves behind larger wound channels when used in an offensive role.",
+    "weight": "18 g",
```

Comment:
18 + 7 (the case weight) matches up fairly well with data on https://www.sellier-bellot.cz/en/products/rifle-ammunition/list/1529/